Formula/t/toxiproxy.rb

```diff
@@ -0,0 +1,54 @@
+class Toxiproxy < Formula
+  desc "TCP proxy to simulate network & system conditions for chaos & resiliency testing"
+  homepage "https://github.com/shopify/toxiproxy"
+  url "https://github.com/Shopify/toxiproxy/archive/refs/tags/v2.6.0.tar.gz"
+  sha256 "17bf7580644a5cf8d6fc2b5a71e8dc7931528838724a6e4dc2bb326731fab4c7"
+  license "MIT"
+
+  depends_on "go" => :build
+  uses_from_macos "ruby" => :build
```

Comment:
```suggestion
  uses_from_macos "ruby" => :test
```

Reply:
```
==> ruby /tmp/toxiproxy-test-20230824-9657-128oi22/hello-homebrew.rb
<internal:/home/linuxbrew/.linuxbrew/Cellar/ruby/3.2.2_1/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require': cannot load such file -- webrick (LoadError)
	from <internal:/home/linuxbrew/.linuxbrew/Cellar/ruby/3.2.2_1/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /tmp/toxiproxy-test-20230824-9657-128oi22/hello-homebrew.rb:1:in `<main>'
```

Let me just start it from the test in a separate thread.